### DISCUSSION TITLE
FINAL Saucedacity/Tenacity 1.3 Release Agenda

### DISCUSSION THREAD

**Discussion by generic-pers0n** [saucedacity/saucedacity#65] [Announcements]
Audacity 3.2 has been released with quite a couple of changes, including VST3 effects support and realtime effects capabilities. On the other hand, we've also been cooking something up.

**I have finally decided on a final Saucedacity/Tenacity 1.3 release agenda. The release agenda goes as follows below.**

# Saucedacity/Tenacity 1.3 Release Agenda
## New
**From Audacity**:
* Clip titles (**in progress**; text rendering works)
* Context menus in the track area and for clips (**ready for merge**)

**From Tenacity**:
* New themes and cursors (**already done**)
* Dynamic compressor (**already done**)
* Translations (**planned**)
* Major build system changes (merging Tenacity's build system; **in progress**)

## Changes
* Reintroduce a way to differentiate our preferences from Audacity's preferences to prevent confusion in our code between legacy versions of Audacity and modern versions of Saucedacity.

* More fixes and cleanups wherever possible (e.g., remove stop-gap solutions, namely `StackAllocator` which has a somewhat misleading name).

## What Could Make It
* Proper JACK support, per #63.

## Removals
None planned.

---

This release agenda outlines what to expect. Something along the way may come up and we might add it to 1.3. Feedback from all, especially contributors, is highly welcome.

**Comment by poyribn** [comment]
i like this release agenda for the same reasons as [the draft version](https://github.com/saucedacity/saucedacity/discussions/62):

> this is a great agenda because it's short.
> 
> especially with the Saucedacity→Tenacity rebranding and the major build system changes, having not much else to test may be needed for stability, especially since Saucedacity is a small project with (as far as i know) no paid (hired) developers yet.

[(link to original comment)](https://github.com/saucedacity/saucedacity/discussions/62#discussioncomment-3682036)
